english/scarab/goddess.md

```diff
@@ -0,0 +1,17 @@
+Alice isn't Alice anymore.  
+
+There is a deeper set of reality to which you just fell through.  
+
+In an instant you understand everything and yet nothing.
+
+You exist in a black room, the walls could be 1 meter or a million kilometers away, you cannot tell.  
+
+The Sentinel is smiling upon you.  At that moment you realize you are not supposed to be there.   It all makes sense now.  
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.